tests/cli/test_cli.py

```diff
@@ -896,6 +896,8 @@ def test_rm(ctx):
     uuid = _run_command([cl, 'upload', test_path('a.txt')])
     _run_command([cl, 'add', 'bundle', uuid])  # Duplicate
     _run_command([cl, 'rm', uuid])  # Can delete even though it exists twice on the same worksheet
+    result = _run_command([cl, 'rm', ''], expected_exit_code=1)  # Empty parameter should give an Usage error
+    check_equals(result, '')
```

Comment:
Is this necessary to check?

Reply:
I thought I could check the info printed to the console this way, which turned out to be not true.